src/open-weather-map.coffee

```diff
@@ -21,13 +23,14 @@
 #   github.com/tan3
 
 weatherURL = process.env.HUBOT_OPEN_WEATHER_MAP_URL or 'http://api.openweathermap.org/data/2.5/weather?q='
+forecastURL = process.env.HUBOT_OPEN_WEATHER_MAP_FORECAST_URL or 'http://api.openweathermap.org/data/2.5/forecast?q='
 cities     = process.env.HUBOT_OPEN_WEATHER_MAP_DEFAULT_CITIES
 apiKey     = process.env.HUBOT_OPEN_WEATHER_MAP_APIKEY
 units      = process.env.HUBOT_OPEN_WEATHER_MAP_UNITS or 'imperial'
 
 module.exports = (robot) ->
   robot.respond /weather help$/i, (msg) ->
-    msg.send 'Usage: hubot weather or hubot weather <city>'
+    msg.send 'Usage: hubot weather or hubot weather <city> or hubot forecast <city for forecast'
```

Comment:
Mind closing the angle bracket here with a `>`? Thanks!

Reply:
Sorry for the typo
Corrected now. Thanks